### ISSUE TITLE
Sample app doesn't work on Chrome OS

### ISSUE THREAD

**Issue by dodalovic** [zeit/next.js#12782]
I tried following this: 

https://nextjs.org/learn/basics/create-nextjs-app/setup

When I issue
```bash
~/me/projects/nextjs-blog$ yarn dev
yarn run v1.22.4
$ next dev
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`
[ ready ] compiled successfully - ready on http://localhost:3000
[ wait ]  compiling ...
[ ready ] compiled successfully - ready on http://localhost:3000
```
Then in browser I get 

This site can’t be reached
localhost refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED

```bash
yarn -v
1.22.4

node -v
v12.16.3

```

**Comment by dodalovic** [comment]
It works with 

```
yarn run dev -H 0.0.0.0
```
maybe it should be documented, or ...?

**Comment by timneutkens** [comment]
Feel free to send a PR!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
